packages/web-components/src/components/table-of-contents/__stories__/README.stories.mdx

```diff
@@ -17,25 +17,36 @@ import contributing from '../../../../../../docs/contributing-license.md';
 
 Here's a quick example to get you started.
 
+> 💡 `bx--tableofcontents__contents` class in the content `<div>` ensures the `<h3>` is visible by navigating to a hashbang URL (e.g. `https://yoursite.com/yourpage#2`).
+
 ##### JS
 
 ```javascript
 import '@carbon/ibmdotcom-web-components/es/components/table-of-contents/table-of-contents';
 ```
 
+##### SCSS
+
+```scss
+@import '@carbon/ibmdotcom-styles/scss/globals/vars';
+@import '@carbon/ibmdotcom-styles/scss/components/tableofcontents/tableofcontents';
+```
+
 ### HTML
 
 ```html
 <dds-table-of-contents>
-  <a name="1" data-title="Section - 1"></a>
-  <h3>Section - 1</h3>
-  <p>Section 1 content</p>
-  <a name="2" data-title="Section - 2"></a>
-  <h3>Section - 2</h3>
-  <p>Section 2 content</p>
-  <a name="3" data-title="Section - 3"></a>
-  <h3>Section - 3</h3>
-  <p>Section 3 content</p>
+  <div class="bx--tableofcontents__contents">
```

Comment:
@asudoh are we replying on adopters to add this wrapper class? Anyway this element can be transcluded instead?

Reply:
> @asudoh are we replying on adopters to add this wrapper class?

Yes. Seems that we need this kind of wrapper class for React, too, given navigating directly to hashbang URL fails there.

> Anyway this element can be transcluded instead?

Do you have any more details here...? Thanks!

Reply:
What I mean is can we add the wrapper div for the user instead of asking them to add it?

Reply:
Thank you for clarifying @jeffchew! I thought of making it automatic, but the problem is that our style (shadow DOM style) doesn't have direct control over the content style, and we allow arbitrary content for `<dds-table-of-contents>`. Therefore it looks like it's more prudent for us to clearly document that user has to use our CSS class to adjust the anchor position. I used below wording as an explanation of what the example code does, but good to hear better wording, etc.:

> 💡 `bx--tableofcontents__contents` class in the content `<div>` ensures the `<h3>` is visible by navigating to a hashbang URL (e.g. `https://yoursite.com/yourpage#2`).